plugins/outputs/cloudwatch/cloudwatch.go

```diff
@@ -332,14 +366,17 @@ func (c *CloudWatch) pushMetricDatumBatch() {
 	}
 }
 
-//sleep some back off time before retries.
+// backoffSleep sleeps some amount of time based on number of retries done.
 func (c *CloudWatch) backoffSleep() {
 	var backoffInMillis int64 = 60 * 1000 // 1 minute
 	if c.retries <= defaultRetryCount {
-		backoffInMillis = int64(backoffRetryBase * math.Pow(2, float64(c.retries)))
+		backoffInMillis = int64(backoffRetryBase * 1 << c.retries)
+		// Adding at most 1 second on each retry for the sake of jitter.
+		backoffInMillis += getJitter(1000)
 	}
 	sleepDuration := time.Millisecond * time.Duration(backoffInMillis)
-	log.Printf("W! %v retries, going to sleep %v before retrying.", c.retries, sleepDuration)
+	log.Printf("W! %v retries, going to sleep %v before retrying.", c.retries,
```

Comment:
since we're already here, would it make sense to include the `cloudwatch` plugin as part of the log message here?

Reply:
Yes.
Will fix.